src/quantum/index.jsx

```diff
@@ -504,65 +513,64 @@ export default class QuantumIndex extends React.Component {
                   item
                   xs={6}
                   key={`page_${sectionId}_${wi}`}
-                  className={(cssRowExtraClasses) ? ` ${cssRowExtraClasses}` : ''}
-                >
+                  className={
+                    cssRowExtraClasses ? ` ${cssRowExtraClasses}` : ''
+                  }>
                   {widget}
                 </Grid>
               );
-            })
-          }
-        </Grid>
-      );
-
-      const stati = toPairs(statusList)
-        .map(([status, count]) => {
-          const desc = statusLabels[status];
-          if (desc && count) {
-            return (
-              <div key={`status-${status}`} className={`header-status header-status-${status}`}>
-                <em>{count}</em>
-                {' '}
-                {desc}
-              </div>
-            );
-          }
-          return null;
+            })}
+          </Grid>
+        );
+        const stati = toPairs(statusList)
+          .map(([status, count]) => {
+            const desc = statusLabels[status];
 
-        })
-        .exists()
-        .toArray();
+            if (desc && count) {
+              return (
+                <div
+                  key={`status-${status}`}
+                  className={`header-status header-status-${status}`}>
+                  <em>{count}</em> {desc}
+                </div>
+              );
+            }
 
-      return (
-        <div key={sectionId}>
-          <h2 className='section-header'>
-            <span>
-              {`${title}`}
-              {more && (
-                <span>
-                  {' ('}
-                  <a href={more}>{'more'}</a>
-                  {')'}
-                </span>
-              )}
-            </span>
-            {stati && ` ${stati}`}
-          </h2>
-          {section}
-        </div>
-      );
+            return null;
+          })
+          .exists()
+          .toArray();
 
-    });
+        return (
+          <div key={sectionId}>
```

Comment:
Here I'm getting this error. 
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md

It doesn't allow us to use the array index as a key. All examples are using `.id` pattern to make it work. 
I did `{sectionId.id}` following the documentation, error removed but it didn't make sense to me. 
The same error is [here](https://github.com/mozilla-frontend-infra/firefox-health-dashboard/blob/master/src/quantum/index.js#L506) also. But using `.id` didnt work here.

@klahnakoski , . @sarah-clements  any thoughts how can I fix it?

Reply:
In this case, this `<div>` is being returned from a `map` operation about 50 lines above:
```js
  const reduced = sections.map(
    ({ title, more, rows, cssRowExtraClasses }, sectionId) => {  ... });
```
the second argument to a map function is the index in the array, which would change if something were added or removed from the array.  That's what the lint is complaining about.

To fix, I'd suggest a similar thing to my previous suggestion: give each section a `key` property that is unique and similar to the title.  So that would mean adding a `key` property to each section in the *long* list of sections above, then passing that to the `<div>`.  Something like

```js
      ...
      {
        cssRowExtraClasses: 'generic-metrics-graphics photon-perf',
        title: 'Performance Metrics',
        key: 'performance-metrics',
        rows: [ ... ]
      }, ...
  ...
  const reduced = sections.map(
    ({ title, key, more, rows, cssRowExtraClasses }, sectionId) => {
      return <div key={key} ... >;
    });
```

Reply:
I created a unique sectionId by combining sectionId + title it then worked and is now merged. 

Your suggestion is good too :) Thanks

Reply:
That sort of "cheats" by hiding the use of the array index.  But in this case, the array doesn't ever change (users can't add or remove things to it while the page is loaded), so that should be fine.  So :+1: